modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessor.java

```diff
@@ -253,6 +253,7 @@ protected void initModels() {
 	protected void process() {
 		_ddmFormLayout.setNextPage(_jsonObject.getInt("nextPage"));
 		_ddmFormLayout.setPreviousPage(_jsonObject.getInt("previousPage"));
+		_ddmFormLayout.setCurrentPage(_jsonObject.getInt("activePage"));
```

Comment:
@linolaoj Can you sort this alphabetically by putting `_ddmFormLayout.setCurrentPage` before `_ddmFormLayout.setNextPage`? Thanks!

Reply:
@carolmariaabb , I'll make this change.